LayoutTests/imported/w3c/web-platform-tests/resources/testdriver-vendor.js

```diff
@@ -263,16 +259,16 @@ window.test_driver_internal.action_sequence = function(sources)
             pointerSource = source;
             break;
         default:
-            return Promise.reject(new Error(`Unknown source type "${action.type}".`));
+            throw new Error(`Unknown source type "${action.type}".`);
```

Comment:
since you're touching this, this should actually be `source.type` (many tests will need a rebaseline, but I think it's worth doing the change).

Reply:
ok, cool. Let's see what comes back..